src/System.IO.FileSystem/tests/Path/IsPathRooted.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Path_IsPathRooted : FileSystemTest
+    {
```

Comment:
This tests doesn't exist, could you please add them instead on PathTests.cs as @danmosemsft mentioned before? 

https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/tests/System/IO/PathTests.cs